kube/public-site-service.yml

```diff
@@ -7,9 +7,9 @@ metadata:
     name: public-site-svc
 spec:
   selector:
-    name: public-site-svc
+    app.kubernetes.io/name: public-site-svc
   type: ClusterIP
   ports:
-    - name: public-prxy-prt
+    - protocol: TCP
```

Comment:
https://kubernetes.io/docs/reference/networking/service-protocols/#protocol-tcp 

TCP is the default -